app/src/main/java/com/mstudio/superheroarch/presentation/overview/CharactersAdapter.kt

```diff
@@ -13,9 +14,10 @@ class CharactersAdapter(
 
     private var items = listOf<CharactersRemoteEntity>()
 
+    @SuppressLint("NotifyDataSetChanged")
     fun updateItems(dataList: List<CharactersRemoteEntity>) {
         items = dataList
-        notifyItemRangeChanged(0, items.size)
+        notifyDataSetChanged()
```

Comment:
still deprecated

Reply:
with notifyitemrangechanged is not working ... for this I decided to use this again 🤔